drape_frontend/gui/gui_text.cpp

```diff
@@ -169,7 +168,7 @@ void StaticLabel::CacheStaticText(std::string const & text, char const * delim,
   ranges.reserve(buffers.size());
 
   float fullHeight = 0.0;
-  float prevLineHeight = 0.0;
+  float prevLineHeight;
```

Comment:
So, in this case better to move declaration in a usage place, inside loop.

Reply:
Right! Thanks.